Assets/MRTK/Core/StandardAssets/FontsSDFTextures/seguisb SDF.asset

```diff
@@ -2909,7 +2909,7 @@ Material:
     - _VertexOffsetY: 0
     - _WeightBold: 0.75
     - _WeightNormal: 0
-    - _ZWrite: 1
+    - _ZWrite: 0
```

Comment:
Thank you for fixing this Vanessa! 

I'm guessing someone enabled depth write on this material to fix a depth based late stage reprojection issue. Based on where this material is being used that may not be an issue anymore? But, it would be good to sanity check where this material is being used (we can use the [dependency tracker](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Tools/DependencyWindow.html)) and view the text renderer on HoloLens with depth buffer sharing enabled (it's on by default in MRTK).

I would only expect issues to only appear with text "floating" out in space without anything writing depth nearby. For example the SectionSubtitles we have in example scenes might begin to "smear" and become unstable if they don't write depth.

Reply:
If issues do crop up, we could duplicate this material and call it out in name as not writing depth, then use it on this UGUI element.

Reply:
You were right @Cameron-Micka . Although I wasn't able to replicate the issue fixed by this [PR](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6474), the zwrite value was changed to 1 to fix a lsr issue. This asset is also being used by a number of scenes in MRTK, so I have duplicated the asset and changed the UGUI prefab from Hand Interaction Example scene.

Reply:
Looks great, thank you @vaoliva!